web-api/src/personalization/routes.rs

```diff
@@ -274,8 +280,10 @@ async fn personalized_documents(
 struct PersonalizedDocumentData {
     id: DocumentId,
     score: f32,
-    #[serde(skip_serializing_if = "HashMap::is_empty")]
-    properties: DocumentProperties,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    properties: Option<DocumentProperties>,
```

Comment:
🔧 this creates an issue as it will not skip serialization for empty properties `Some(json!({}))`, but only if `include_properties = false`. this also contradicts what we say in the frontoffice docs: https://github.com/xaynetwork/xayn_discovery_engine/blob/main/web-api/openapi/front_office.yaml#L182

so we could change the docs to account for the new behavior, but i think it would be better to change the serialization check to `option is_none || inner is_empty`.